src/libs/ReportUtils.js

```diff
@@ -1811,20 +1817,16 @@ function isUnreadWithMention(report) {
  * @returns {boolean}
  */
 function hasOutstandingIOU(report, iouReports) {
-    if (!report || !report.iouReportID || _.isUndefined(report.hasOutstandingIOU)) {
-        return false;
-    }
```

Comment:
Is it safe to remove this?

Reply:
It's only used in 1 place: [shouldReportBeInOptionList](https://github.com/Expensify/App/blob/c370ea925847dcf68eb48df0ed984eb8e4112ee1/src/libs/ReportUtils.js#L1930). I wanted to force displaying the iou request on the LHN when IOU is outstanding or is waiting on bank account, but It actually didn't work, so I might as well revert it to how it was. And maybe we do it on purpose to show the IOU details on the LHN only once you click on the message from the parent report? ... but then we'd need to show the red dot on the parent, no? .. I think I'm a bit confused on what we should expect on the LHN.

Reply:
There is a incomplete implementation yet where the expense/iou reports are not showing from the GetChats call so when you log out and log back in you have to click on the report preview to load it